app/code/Codifi/CustomerRequest/Console/PackCustomerNotes.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\CustomerRequest\Console;
```

Comment:
The recommended location is Console\Command
https://devdocs.magento.com/guides/v2.4/extension-dev-guide/cli-cmds/cli-howto.html#cli-sample